src/App.tsx

```diff
@@ -10,6 +12,8 @@ const App = () => (
     <Switch>
       <Route exact path="/" component={Welcome} />
       <Route path="/dashboard" component={Home} />
+      <Route path="/signIn" component={SignIn} />
+      <Route path="/signUp" component={SignUp} />
```

Comment:
I would recommend keeping all URLs lower case fully. Programmers are used to camel case but general users like something simple.

Reply:
changed to /signin and /signup